src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/servicebus/PaymentMessageProcessor.java

```diff
@@ -22,7 +22,7 @@
 
 
 @Service
-@Profile("!nosb") // do not register for the nosb (test) profile
+@Profile("!functional")
```

Comment:
This change will make `PaymentMessageProcessor ` available when `nosb` profile is used.
Was that your intention?

Reply:
nosb profile is renamed to functional